appservice/src/createAppService/SiteNameStep.ts

```diff
@@ -22,7 +23,7 @@ const siteNamingRules: IAzureNamingRules = {
 
 export class SiteNameStep extends AzureNameStep<IAppServiceWizardContext> {
     public async prompt(context: IAppServiceWizardContext): Promise<void> {
-        const client: WebSiteManagementClient = await createWebSiteClient(context);
+        const client = await createGenericClient(context, context);
```

Comment:
Was it intentional to pass context in twice?

Reply:
Yep, in this case, context also acts as credentials because `IAppServiceWizardContext` extends `ISubscriptionActionContext`.

```
if (clientInfo && 'credentials' in clientInfo) {
        credentials = clientInfo.credentials;
        baseUri = clientInfo.environment.resourceManagerEndpointUrl;
    } else {
        credentials = clientInfo;
    }
```

This code block in `createGenericClient` handles setting that.